mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -538,7 +540,12 @@ public void runVoid() throws MailboxException {
                     }
                 }, true);
 
+            int mailboxIdCount = mailboxIds.size();
+            if (mailboxIdCount > 0) {
+                return Optional.of(mailboxIds.get(mailboxIdCount - 1));
```

Comment:
See @mbaechler comment : It is the case, and I updated code to make it blattent.